addons/website_sale_digital/controllers/main.py

```diff
@@ -34,7 +34,7 @@ def portal_order_page(self, order_id=None, **post):
         if not 'sale_order' in response.qcontext:
             return response
         order = response.qcontext['sale_order']
-        invoiced_lines = request.env['account.move.line'].sudo().search([('invoice_id', 'in', order.invoice_ids.ids), ('invoice_id.state', '=', 'paid')])
+        invoiced_lines = request.env['account.move.line'].sudo().search([('move_id', 'in', order.invoice_ids.ids), ('move_id.state', '=', 'paid')])
```

Comment:
`'move_id.invoice_payment_state', '=', 'paid'` as `paid` is no longer a valid key inside `state`

Reply:
Thanks :)

I have updated the code :rocket: